src/components/Launch/LaunchWorkflowForm/inputConverters.ts

```diff
@@ -1,62 +1,166 @@
 import { dateToTimestamp, millisecondsToDuration } from 'common/utils';
 import { Core } from 'flyteidl';
 import * as Long from 'long';
+import * as LossLessJSON from 'lossless-json';
 import { utc as moment } from 'moment';
-import { InputType } from './types';
+import {
+    InputProps,
+    InputType,
+    InputTypeDefinition,
+    InputValue
+} from './types';
 
-function booleanToLiteral(value: string): Core.ILiteral {
-    return { scalar: { primitive: { boolean: Boolean(value) } } };
+function losslessReviver(key: string, value: any) {
+    if (value && value.isLosslessNumber) {
```

Comment:
Does this handle a value of `0`?

Reply:
I see why you're asking (0 is falsey). Your intuition is correct, but it should be fine in this case, as the behavior is just to return `value` directly. That would result in a `number` and is the desired behavior.